mainform.py

```diff
@@ -53,20 +59,23 @@ def changeLangF(self):
         self.changeLang('translation/fr_translation.qm') #установление языка
 
     def changeLang(self, lang):
+        self.lang=lang
         try:
             self.app.removeTranslator(self.translator) #модель перевода удаляется
             self.app.removeTranslator(self.common_translator)
         except:
             pass
-        if lang != 'e': #если выбран не английский
+        if lang != 'en': #если выбран не английский
             self.translator = QtCore.QTranslator() #вызывается переводчик
```

Comment:
У меня Qt Linguist
--
Envoyé de l'application Mail.Ru pour Android vendredi, 09 septembre 2016, 03:51PM +03:00 de Cadovvl < notifications@github.com> :

>In  mainform.py :
>>          try:
>              self.app.removeTranslator(self.translator) #модель перевода удаляется
>              self.app.removeTranslator(self.common_translator)
>          except:
>              pass
> -        if lang != 'e': #если выбран не английский
> +        if lang != 'en': #если выбран не английский
>              self.translator = QtCore.QTranslator() #вызывается переводчик
>А напомни, каким переводчиком пользуется QT?
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .